source/infrastructure/bin/aws-instance-scheduler.ts

```diff
@@ -56,4 +57,5 @@ new AwsInstanceSchedulerRemoteStack(app, 'aws-instance-scheduler-remote', {
     solutionVersion: SOLUTION_VERSION,
     appregApplicationName: APP_REG_APPLICATION_TYPE,
     appregSolutionName: APP_REG_SOLUTION_NAME
-});
\ No newline at end of file
+});
+new PipelineStack(app, 'testing-pipeline');
```

Comment:
Isn't this the name of the stack in cfn? Shouldn't it be a bit more descriptve? e.g. `InstanceSchedulerTestingPipeline`?

Reply:
it appears to be applied as a prefix before a CDK Pipeline hash that keeps it unique. a more specific name is probably a good idea.  I had originally been trying to keep it generic for reusability, but I don't think this pipeline code is quite there yet